### ISSUE TITLE
Interference with app redirect

### ISSUE THREAD

**Issue by urherenow** [rashevskyv/dbi#101]
Just used DBI for the first time because tinfoil was locking up on a couple of new(er) files. The install of the first went ok, although it appeared to be stuck on the PC side (like an issue already posted here). Second file, having rebooted and starting over, worked as expected in all counts.

The problem: I don't know if applet mode is needed to run this, but in both cases, I launched HBL using the redirect to my inserted Metroid Dread cart. After installing a file with DBI (using MTP responder), and closing the "Metroid Dread" app, Trying to launch the real Metroid Dread would crash, saying the software could not be read. Same when trying to hold R to open HBL again. a Reboot fixes the issue. This happened twice in a row, and I am able to close it and open as the Game, or as a redirected HBL repeatedly. This issue only pops up after installing a file with DBI.

I hope I've explained this well enough...

**Comment by Masamune3210** [comment]
I'd wager this isnt a issue with DBI likely, if its also failing trying to run the taken over app as well as when not taken over with the same error. Maybe a issue with taking over cart apps?

**Comment by urherenow** [comment]
Nope. Bet you anything it's an issue with the MTP responder. Just did several more scenario tests. This ONLY occurs, when starting the MTP Responder. I don't even need to install anything. When starting the MTP responder, one of the steps on the screen, is checking the game card. This is where the bug is. I'm quite sure you can duplicate this yourself, if you own a game cart, and do the title takeover with it.

**Comment by rashevskyv** [comment]
Applet mod is recommend for DBI

**Comment by urherenow** [comment]
Same thing. 

Just launched in applet mode, started MTP responder, closed, and my game cart can't open. The "Checking Game Card" is clearly messing something up.

**Comment by Masamune3210** [comment]
You are right, just tried it with a cart and had the same issue happen

**Comment by lowburger** [comment]
You can make MTP gamecard check to false in dbi config file for seeing if it's cause of issue.

**Comment by Otorhin** [comment]
Не умею в англ, но оставлю также
После запуска MTP Responder для запуска игр с картриджа, требуется его переставить в консоль или перезагрузить её. Не зависит от того как запустить DBI

**Comment by lowburger** [comment]
> Не умею в англ, но оставлю также После запуска MTP Responder для запуска игр с картриджа, требуется его переставить в консоль или перезагрузить её. Не зависит от того как запустить DBI https://youtu.be/BlHPsG3kY2c

Where we find DBI version 373?

**Comment by duckbill007** [comment]
> > Не умею в англ, но оставлю также После запуска MTP Responder для запуска игр с картриджа, требуется его переставить в консоль или перезагрузить её. Не зависит от того как запустить DBI https://youtu.be/BlHPsG3kY2c
> 
> Where we find DBI version 373?

Нигде. Это частный билд с индивидуальными фичами и исправлениями.
